apiserver/uniter/uniter.go

```diff
@@ -1564,8 +1581,10 @@ func (u *UniterAPIV3) AddMetricBatches(args params.MetricBatchParams) (params.Er
 	return result, nil
 }
 
+// TODO(wpk) Uniter.NetworkConfig API is obsoleted by Uniter.NetworkInfo
 // NetworkConfig returns information about all given relation/unit pairs,
 // including their id, key and the local endpoint.
+// It's not included in APIv6
 func (u *UniterAPIV3) NetworkConfig(args params.UnitsNetworkConfig) (params.UnitNetworkConfigResults, error) {
```

Comment:
Don't you also need to create a 
func (u *UniterAPIV3) NetworkInfo(InvalidParameters, and stuff like that)

So that we don't expose NetworkInfo in the v3/4/5 APIs?
Do we care?